deploy/Centos/ntnx-csi-node.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       labels:
         app: csi-node-ntnx-plugin
     spec:
-      serviceAccount: csi-node-ntnx-plugin
+      serviceAccountName: csi-node-ntnx-plugin
```

Comment:
Why do you have to change serviceAccount to servviceAccountName ?

Reply:
serviceAccount is deprecated, need to use serviceAccountName instead.

Ref: ds.spec.template.spec

DeprecatedServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.

Reply:
Thanks for the info.